code/game/machinery/doors/firedoor.dm

```diff
@@ -118,6 +117,9 @@
 		. += span_notice("It is <i>welded</i> shut. The floor bolts have been locked by <b>screws</b>.")
 	else
 		. += span_notice("The bolt locks have been <i>unscrewed</i>, but the bolts themselves are still <b>wrenched</b> to the floor.")
+	if(base_icon_state == "sus")
+		. += span_warning("This one looks a bit sus...")
```

Comment:
and this one, this isnt a temporary state, its a meme, the meme should be self contained